.evergreen/evergreen.yml.in

```diff
@@ -76,6 +76,579 @@ const platformDetails = platformToDetails[platform];
   }
 }
 
+const { RELEASE_PACKAGE_MATRIX } = require('../config/release-package-matrix');
+
+const ALL_E2E_TEST_BUILD_VARIANTS =  [
```

Comment:
I admit this change may have gone a bit overboard 🙈  but hopefully for the better? I had a lot of trouble navigating in yaml and seemed like a lot of the variants were hardcoded and I need to extract the openssl configurations through something like a script (which I have included in the PR description) and I figured I'd keep it as a standardized array rather than more YAML). Might be worth extracting it to a js file like the release matrix. 

Also open to changing the structure if anything could be worded better; pretty simple to apply the changes as needed.

Reply:
Yep – I definitely wouldn't mind extracting this into its own file 👍